std/algorithm/iteration.d

```diff
@@ -2973,6 +2977,389 @@ template fold(fun...) if (fun.length >= 1)
     static assert(is(typeof(arr.fold!((a, b) => a)(1))));
 }
 
+/++
+Similar to $(LREF fold), but returns a range containing the successive reduced
+values. The call $(D cumulativeFold!(fun)(range, seed)) first assigns $(D seed)
+to an internal variable $(D result), also called the accumulator.
+The returned range contains the values $(D result = fun(result, x)) lazily
+evaluated for each element $(D x) in $(D range). Finally, the last element has
+the same value as $(D fold!(fun)(seed, range)).
+The one-argument version $(D cumulativeFold!(fun)(range)) works similarly, but
+it returns the first element unchanged and uses it as seed for the next
+elements.
+This function is also known as `partial_sum`, `accumulate`, `prefix_sum`,
+`scan`, or `cumulative_sum`.
+
+Returns:
+    a range containg the consecutive reduced values. If there is more than one
```

Comment:
Looking over the produced documentation: http://dtest.thecybershadow.net/artifact/website-d6d12c1c1afc9a7a52b48c36a2c67cf74b2cf7ba-c09403ae36e48098141b671f02e526c5/web/library-prerelease/std/algorithm/iteration/cumulative_fold.html

It is best to NOT consider "Returns" as the start of a sentence. Just write something like this: "The function returns a range containing..."